test/test_setup.py.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I don't really know a good way to run this, but here ya go, a little bash script test of buildability. This was really important for getting all the `pkg_resources` stuff right.